configs/tmqm/default.py

```diff
@@ -0,0 +1,103 @@
+"""Defines the default training configuration."""
+
+import ml_collections
+
+
+def get_config() -> ml_collections.ConfigDict:
+    """Get the default training configuration."""
+    config = ml_collections.ConfigDict()
+
+    # Dataset.
+    config.dataset = "tmqm"
+    config.fragment_logic = "nn"
+    config.train_on_split_smaller_than_chunk = False
+    config.root_dir = None
+    config.num_train_molecules = 69000
+    config.num_val_molecules = 9000
+    config.num_test_molecules = 8665
+    config.shuffle_datasets = True
+    config.infer_edges_with_radial_cutoff = True
+    config.radial_cutoff = 5.0
+    config.max_targets_per_graph = 4
+    config.continuous = False
```

Comment:
config.continuous is too vague, add this instead to the config.target_position_predictor,
maybe as `config.target_position_predictor.continuous_radius`